### PR TITLE
Add runsOnRaspberryPi

### PR DESCRIPTION
**Pull request by FDelporte**
Extra boardinfo method to easily check if an app is running on a Raspberry Pi board.

Maybe we can still include this in V2.6.1?

### CODE REVIEW

**Review by taartspi — APPROVED** [review]
Looks good.    And i remember earlier conversations that linuxfs is expected to load ok on a foreign platform so this was not needed.

File: pi4j-core/src/main/java/com/pi4j/boardinfo/util/BoardInfoHelper.java [hunk 1]
```diff
@@ -64,6 +64,10 @@ public static boolean usesRP1() {
         return instance.boardInfo.getBoardModel() == BoardModel.MODEL_5_B;
     }
 
+    public static boolean runsOnRaspberryPi() {
```
Comment (eitch) [inline pi4j-core/src/main/java/com/pi4j/boardinfo/util/BoardInfoHelper.java]: maybe we want to name it differently: runningOnRaspberryPi() ?

Comment (FDelporte) [reply pi4j-core/src/main/java/com/pi4j/boardinfo/util/BoardInfoHelper.java]: good point! will change

Comment (FDelporte) [reply pi4j-core/src/main/java/com/pi4j/boardinfo/util/BoardInfoHelper.java]: is done

**Review by eitch — APPROVED** [review]

**Pull request merged**